src/javascript/app_2/pages/trading/components/elements/drawer.jsx

```diff
@@ -132,8 +171,89 @@ class DrawerItem extends React.PureComponent {
     }
 }
 
+
+const DrawerHeader = ({
+    alignment,
+    closeBtn,
+}) => {
+    const drawer_header_class = classNames('drawer-header', alignment);
+    return (
+        <React.Fragment>
+            {alignment && alignment === 'right' ?
+                <div className={drawer_header_class}>
+                    <div className='icons btn-close' onClick={closeBtn}>
+                        <img src={Url.urlForStatic('images/trading_app/common/close.svg')} alt='Close' />
+                    </div>
+                </div>
+            :
+                <div className={drawer_header_class}>
+                    <div className='icons btn-close' onClick={closeBtn}>
+                        <img src={Url.urlForStatic('images/trading_app/common/close.svg')} alt='Close' />
+                    </div>
+                    <div className='icons brand-logo'>
+                        <img src={Url.urlForStatic('images/trading_app/header/binary_logo_dark.svg')} alt='Binary.com' />
+                    </div>
+                </div>
+        }
+        </React.Fragment>
+    );
+};
+const DrawerFooter = ({
+    alignment,
+}) => (
+    <React.Fragment>
+        {alignment && alignment === 'right' ?
+        null
+        :
+        <div className='drawer-footer' />
```

Comment:
Could be:
```js
{ alignment !== 'right' &&
        <div className='drawer-footer' />
}
```

I'd suggest not to use alignment to handle visibility of the footer, prefer to explicitly pass the proper argument.

Reply:
Agreed, is it fine if I just call the `drawer-footer` directly to the left aligned component?

Reply:
I think each common component should handle its regarding children by itself, then the component which uses the `<Drawer />` only needs to pass properties, the rest is handled inside.